filesystem/fat32/fat32.go

```diff
@@ -572,7 +572,7 @@ func (fs *FileSystem) getClusterList(firstCluster uint32) ([]uint32, error) {
 		switch {
 		case fs.table.isEoc(newCluster):
 			complete = true
-		case cluster <= 2:
+		case cluster < 2:
```

Comment:
@deitch this change is critical because block 2 can be a chain

Reply:
Makes sense.